src/main/java/Ui.java

```diff
@@ -0,0 +1,102 @@
+import java.io.IOException;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+
+    private TaskList tasks;
+    private Storage storage;
+
+    /**
+     * Creates a Ui object to interact with the user.
+     * @param tasks TaskList containing the user's tasks.
+     * @param storage Storage for loading and saving the TaskList to.
+     */
+    public Ui(TaskList tasks, Storage storage) {
+        this.tasks = tasks;
```

Comment:
Your Ui class should not be doing work on the TaskList and Storage, the Ui class should purely be used for output messages. Consider abstracting the work into a separate class. :)